### DISCUSSION TITLE
请问如何在openwrt源码修改，让编译好的固件版本处自动添加编译日期？

### DISCUSSION THREAD

**Discussion by netlovehf** [immortalwrt/immortalwrt#1473] [Q&A]
请问如何在openwrt源码修改，让编译好的固件版本处自动添加编译日期？
![2024-09-19_1811](https://github.com/user-attachments/assets/d2a021fb-5de9-45df-a568-a487d6952a31)

**Comment by smallprogram** [comment]
看了看源码，在这里：

https://github.com/immortalwrt/luci/blob/2ff52942e558d667e1eb8e68586bea014c67745b/modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/10_system.js#L87

后边加上你要加的内容，例如：
`_('Firmware Version'), (L.isObject(boardinfo.release) ? boardinfo.release.description + ' / ' : '') + (luciversion || '') + ' (2024-10-01)', `

如果在编译时加上当前编译的日期，可以写个bash脚本，在编译前添加，这样每次编译就会生成编译时的日期

**Reply by netlovehf** [reply]
> 看了看源码，在这里：
> 
> https://github.com/immortalwrt/luci/blob/2ff52942e558d667e1eb8e68586bea014c67745b/modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/10_system.js#L87
> 
> 后边加上你要加的内容，例如： `_('Firmware Version'), (L.isObject(boardinfo.release) ? boardinfo.release.description + ' / ' : '') + (luciversion || '') + ' (2024-10-01)', `
> 
> 如果在编译时加上当前编译的日期，可以写个bash脚本，在编译前添加，这样每次编译就会生成编译时的日期

我下载的源码是这里下载的https://github.com/immortalwrt/immortalwrt.git，这里怎么修改的

**Reply by smallprogram** [reply]
我说的是luci这个feed
